canonical-web-and-design/jaas.ai
# Refactor modals JS to a module

## Done

Refactor and simplify modals JS to a module so it can be bundled and remove extra HTTP request

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029
- Check experts pages at;

/experts/omnivector
/experts/tengu
/experts/spicule

...and verify clicking "Get in touch" on each will open modal window with contact form.
